code/game/machinery/_machines_base/machinery.dm

```diff
@@ -329,7 +329,14 @@ Class Procs:
 		text = "\The [src] pings."
 
 	state(text, "blue")
-	playsound(src.loc, 'sound/machines/ping.ogg', 50, 0)
+	playsound(src.loc, 'sound/machines/ping.ogg', 50, FALSE)
+
+/obj/machinery/proc/buzz(text=null)
```

Comment:
`text` without `=null` is equivalent.

Reply:
Probably some leftover from copy pasting ping..